chat/src/main/java/bisq/chat/notifications/ChatNotificationService.java

```diff
@@ -386,7 +395,9 @@ private <M extends ChatMessage> ChatNotification createNotification(String id, C
     }
 
     private void maybeSendSystemNotification(ChatNotification chatNotification) {
-        if (!isApplicationFocussed && isReceivedAfterStartUp(chatNotification)) {
+        if (!isApplicationFocussed &&
+            isReceivedAfterStartUp(chatNotification) &&
+            !isExpired(chatNotification)) {
```

Comment:
This will prevent the system notification to be sent. However, we only send system notifications of messages received while using the app so this seems redundant. Have you tested it?
Also, have you tested that the badge is behaving correctly as well? It shouldn't show the badge number for messages which have expired.